gothemis/compare/compare.go

```diff
@@ -77,6 +77,13 @@ const (
 
 // Errors returned by Secure Comparator.
 var (
+	ErrAppendSecret  = errors.New("Failed to append secret")
+	ErrCreateObj     = errors.New("Failed to create comparator object")
```

Comment:
```suggestion
	ErrCreateCmpObj     = errors.New("Failed to create comparator object")
```

That's an invitation to discuss: `ErrCreateObj` has same name, but different error log for creating comparator object, and secure session object. 

Should they be same errors (`ErrCreateObj`) or different (`ErrCreateCmpObj`, `ErrCreateSSObj`)?

Reply:
why not to use full name `ErrCreateComparatorObject`?

Reply:
Well, `ErrCreateComparatorObject` and `ErrCreateSecureSessionObject` then.

Similarly, `ErrDestroyComparatorObject` and `ErrDestroySecureSessionObject`